src/module-builder/get-unique-filename.ts

```diff
@@ -1,11 +1,11 @@
-import uuid from 'shortid'
+import {nanoid} from 'nanoid'
 import fileExtension from 'file-extension'
 
 export const getUniqueFilename = (fullFilename: string) => {
   // filename with no extension
   const filename = fullFilename.replace(/\.[^/.]+$/, '')
   // remove stuff s3 hates
-  const scrubbed = `${filename}-${uuid.generate()}`
+  const scrubbed = `${filename}-${nanoid()}`
```

Comment:
Looking into shortid, I think the default was a 7 character identifier, so for consistency:

```suggestion
  const scrubbed = `${filename}-${nanoid(7)}`
```